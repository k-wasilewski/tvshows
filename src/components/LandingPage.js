import React, { Component } from 'react';
import '../css/App.css';
import Search from './Search';
import Results from './Results';

class LandingPage extends Component {
    render() {
        return (
            <div className="App">
                <Search />
                <Results />
            </div>
        );
    };
};

export default LandingPage;
