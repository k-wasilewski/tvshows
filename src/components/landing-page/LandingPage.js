import React, { Component } from 'react';
import Search from './search/Search';
import Results from './results/Results';

class LandingPage extends Component {
    render() {
        return (
            <React.Fragment>
                <Search />
                <Results />
            </React.Fragment>
        );
    };
};

export default LandingPage;
