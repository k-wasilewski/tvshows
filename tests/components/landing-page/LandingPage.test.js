import React from 'react';
import renderer from 'react-test-renderer';
import store from "../../../src/redux/store";
import {BrowserRouter} from "react-router-dom";
import LandingPage from "../../../src/components/landing-page/LandingPage";
import {Provider} from "react-redux";

describe("LandingPage rendering specification", () => {
    it('LandingPage is rendered', () => {
        const component = renderer.create(
            <Provider store={store}>
                <BrowserRouter>
                    <LandingPage />
                </BrowserRouter>
            </Provider>
        );
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});