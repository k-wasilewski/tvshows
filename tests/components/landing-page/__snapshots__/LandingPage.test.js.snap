// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage rendering specification LandingPage is rendered 1`] = `
Array [
  <form
    className="makeStyles-dashboard-12"
    id="searchForm"
    onSubmit={[Function]}
  >
    <span
      className="makeStyles-errorMsg-6"
    >
      
    </span>
    <input
      className="makeStyles-input-15"
      id="searchFormInput"
      onChange={[Function]}
      type="text"
    />
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedSizeSmall MuiButton-sizeSmall"
      disabled={false}
      id="searchFormSubmitBtn"
      onBlur={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="submit"
    >
      <span
        className="MuiButton-label"
      >
        Szukaj
      </span>
    </button>
    <button
      className="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedSizeSmall MuiButton-sizeSmall"
      disabled={false}
      id="searchFormResetBtn"
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      tabIndex={0}
      type="button"
    >
      <span
        className="MuiButton-label"
      >
        Resetuj wyniki
      </span>
    </button>
    <br />
    <h2
      className="MuiTypography-root makeStyles-query-7 MuiTypography-body1"
    />
  </form>,
  <div
    className="MuiCircularProgress-root MuiCircularProgress-colorPrimary MuiCircularProgress-indeterminate"
    role="progressbar"
    style={
      Object {
        "height": 40,
        "width": 40,
      }
    }
  >
    <svg
      className="MuiCircularProgress-svg"
      viewBox="22 22 44 44"
    >
      <circle
        className="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate"
        cx={44}
        cy={44}
        fill="none"
        r={20.2}
        strokeWidth={3.6}
        style={Object {}}
      />
    </svg>
  </div>,
]
`;
